src/constants/DefaultFields.ts

```diff
@@ -13,6 +13,9 @@ export enum DefaultFieldName {
   PREFERRED_CONTACT = "preferred_comms",
   PREFERRED_NUMBER = "preferred_number",
   WORK_NUMBER = "work_number",
+  ENROLLED = "enrolled",
+  CURRENT_CLASS = "current_class",
+  STATUS = "status",
```

Comment:
in the future, it would be nice to sort these alphabetically! don't worry about it here though since this is refactored in #33